src/components/LanguageSwitch/LanguageSwitch.tsx

```diff
@@ -1,28 +1,16 @@
-import React, { MouseEvent, useCallback, useState, useRef, useContext } from 'react';
+import React, { useCallback, useContext } from 'react';
 import LanguageIcon from '@material-ui/icons/Language';
-import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import i18next, { TFunction } from 'i18next';
 import { useTranslation } from 'react-i18next';
 import { withStyles } from '@material-ui/core/styles';
-import ClickAwayListener from '@material-ui/core/ClickAwayListener';
-import Grow from '@material-ui/core/Grow';
-import Popper from '@material-ui/core/Popper';
-import MenuList from '@material-ui/core/MenuList';
 import styled from '@emotion/styled';
 
 import { Language } from '../../../i18n/config';
 import ThemeContext from '../../design-tokens/ThemeContext';
-import Paper from '../../muiComponents/Paper';
 import MenuItem from '../../muiComponents/MenuItem';
-import Button from '../../muiComponents/Button';
-import Tooltip from '../../muiComponents/Tooltip';
-import Divider from '../../muiComponents/Divider';
-import Box from '../../muiComponents/Box';
 import { Theme } from '../../design-tokens/theme';
-import Link from '../Link';
 import Icon from '../Icon';
-
-const VERDACCIO_UI_GITHUB_REPOSITORY = 'https://github.com/verdaccio/ui';
```

Comment:
Now that we have  the "translation" section in our docs, I believe it is fine to remove it

Reply:
Ok to me.